# message for publishing position (cartesian or gps coordinates)

float32 x
float32 y

float64 latitude
float64 longitude
float64 altitude
float64[9] position_covariance



